bin/gbuild

```diff
@@ -46,6 +46,12 @@ def build_one_configuration(suite, arch, build_desc)
       ENV["LXC_SUITE"] = suite
   end
 
+  if ENV["USE_DOCKER"] and build_desc["distro"].eql? "centos"
+    ontarget_root_extra = "-w /root"
```

Comment:
what happens if we don't supply this argument?

Reply:
On newer versions of docker there appears to be some sort of security setting that causes the command (without the `-w root`) to fail because of bad permissions on /home/centos (which is weird, of course, since the login is `root`).  This showed up on a coworker's Arch system -- but only when using a centos-based container.

Similar reports:

- https://rmoff.net/2018/11/30/logging-in-as-root-on-oracle-database-docker-image/
- https://github.com/coderaiser/cloudcmd/issues/182
- http://www.javamonamour.org/2018/10/more-jira-on-docker.html